code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -141,6 +141,9 @@ GLOBAL_LIST_EMPTY(PDAs)
 	. = ..()
 	if(!equipped)
 		if(user.client)
+			//SKYRAT EDIT ADDITION BEGIN - CUSTOMIZATION
+			ttone = trim(user.client.prefs.read_preference(/datum/preference/text/pda_ringer), 20)
```

Comment:
I feel like this should be sanitized, I feel like someone clever could do some HTML injection with this.

Reply:
I tried putting in some html stuff through char prefs and it didn't do anything. Then again I'm not that clever. In any case, its sanitized now!